### PR TITLE
Ret 2954 Provide something Tribunal asked for - Respond to a notification (ECM)

### PR DESCRIPTION
**Pull request by paul-peasegood**
**Before creating a pull request make sure that:**

Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RET-2954

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: definitions/json/CaseEventToFields.json [hunk 1]
```diff
@@ -7036,6 +7038,122 @@
...
+    "DisplayContext": "MANDATORY",
+    "PageID": 2,
+    "PageDisplayOrder": 2,
+    "PageFieldDisplayOrder": 6,
+    "ShowSummaryChangeOption": "Y"
+  },
+  {
+    "CaseTypeID": "ET_EnglandWales",
+    "CaseEventID": "respondNotification",
+    "CaseFieldID": "respondNotificationResponseRequried",
```
Comment (JackReevies) [inline definitions/json/CaseEventToFields.json:7085]: `respondNotificationResponseRequired` spelling

File: definitions/json/EnglandWales Scrubbed.json [hunk 2]
```diff
@@ -6281,6 +6281,54 @@
...
+    "DisplayOrder": 1
+  },
+  {
+    "ID": "respondNotificationIsThis",
+    "ListElement": "Request",
+    "ListElementCode": "Request",
+    "DisplayOrder": 2
+  },
+  {
+    "ID": " fl_respondNotificationResponseRequried",
```
Comment (JackReevies) [inline definitions/json/EnglandWales Scrubbed.json:6297]: `fl_respondNotificationResponseRequired`

Comment (paul-peasegood) [reply definitions/json/EnglandWales Scrubbed.json]: fixed

File: definitions/json/CaseEventToFields.json (hunk 1 above)
Comment (paul-peasegood) [reply definitions/json/CaseEventToFields.json]: fixed

**Review by JackReevies — APPROVED** [review]

**Review by ikirsanov — APPROVED** [review]

**Pull request merged**